### PR TITLE
updated the error message

### PR DESCRIPTION
**Pull request by gauravagrwal**
# PR for issue
#1122 

## What is being addressed

- Updating the error message when the workspace service template is not found

## How is this addressed

- Updated the `strings.py` file where all the error messages are listed,
- and updated the `workspace_service_templates.py` and `workspace_templates.py` which were referencing that error message.

### CODE REVIEW

File: api_app/api/routes/workspace_templates.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def get_current_template_by_name(template_name: str, template_repo: ResourceTemp
         template = template_repo.get_current_template(template_name, resource_type, parent_service_template_name)
         return template_repo.enrich_template(template)
     except EntityDoesNotExist:
-        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=strings.TEMPLATE_DOES_NOT_EXIST)
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=strings.WORKSPACE_SERVICE_TEMPLATE_DOES_NOT_EXIST)
```
Comment (marrobi) [inline api_app/api/routes/workspace_templates.py:25]: I believe this one needs to remain as `resource_type: ResourceType` might be of a different type.

**Review by marrobi — CHANGES_REQUESTED** [review]

**Comment by marrobi** [comment]
@gauravagrwal thank you for the PR. Believe a small change is required. Thanks.

File: api_app/api/routes/workspace_templates.py (hunk 1 above)
Comment (gauravagrwal) [reply api_app/api/routes/workspace_templates.py:25]: okay, so both error messages(`TEMPLATE_DOES_NOT_EXIST` and `WORKSPACE_SERVICE_TEMPLATE_DOES_NOT_EXIST`) will be there. right?

Comment (marrobi) [reply api_app/api/routes/workspace_templates.py:25]: In this place just `TEMPLATE_DOES_NOT_EXIST` as we do not know the resourceType. Ideally the `resourceType` passed in would be substituted into the error but happy with either for now.

**Comment by marrobi** [comment]
@gauravagrwal all looks good. We have an issue with our end to end tests, will get that sorted then rerun your PR test and merge. Thanks for the contribution.

**Review by martinpeck — APPROVED** [review]
LGTM. Thanks

**Pull request merged**